grails-app/views/purchaseOrder/_showOrderItems.gsp

```diff
@@ -509,7 +507,7 @@
                * TODO here, we may need to elsewhere -- should we escape all
                * TODO localized strings? Further discussion at OBGM-343.
                */
-              if ($("#validationCode").val() == 'WARN' && !confirm(_.escape(htmlDecode("${g.message(code: 'orderItem.warningSupplier.label')}")))) {
+              if ($("#validationCode").val() == 'WARN' && !confirm(htmlDecode("${g.message(code: 'orderItem.warningSupplier.label')}"))) {
```

Comment:
Importing the lodash library was throwing error. After removing it I found there is an reference to global UMD variable. After checking how this message looks like I found that, it's broken:
![image](https://github.com/openboxes/openboxes/assets/83239466/28b957d9-db82-4c0b-8882-806a130c4af2)
There was a [ticket](https://pihemr.atlassian.net/browse/OBPIH-4561?atlOrigin=eyJpIjoiYjY5YzJlYjc4ZWQ4NDBhNWI5MzU5YzQzMjEwZmQzZmQiLCJwIjoiamlyYS1zbGFjay1pbnQifQ) about the quotes.
After removing this method from lodash I found that this message looks correctly:
![image](https://github.com/openboxes/openboxes/assets/83239466/b1477550-0e92-4f1c-841c-738fdd98c74f)
In the comment above this code is a comment about further discussion / investigation about it.
Here is the [ticket](https://pihemr.atlassian.net/browse/OBGM-343?atlOrigin=eyJpIjoiZDM0ZTJhZTJiNjJhNDc0N2E1NzViZmIwZDFjZDdlMGMiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)